scripts/falco-driver-loader

```diff
@@ -216,54 +217,56 @@ load_kernel_module() {
 		exit 1
 	fi
 
-	echo "* Unloading ${DRIVER_NAME} module, if present"
+	echo "* Unloading the Falco module, if present"
```

Comment:
I would leave DRIVER_NAME here, we can discuss this change in another PR since it's not the scope of this one, plus i don't believe this change is useful as it prevents users from reusing this scripts for using multiple Falco instances on the same host.

Reply:
Same for all the other places where DRIVER_NAME was replaced

Reply:
I agree on the guidelines in general but this is the name of the module rather than the name of the project and the module is called `falco` rather than `Falco` now

Reply:
Yup, no problem at all. I'll revert ASAP